app/controllers/items/new.js

```diff
@@ -177,6 +192,32 @@ export default GoodcityController.extend(
       return detailAttributes;
     },
 
+    paramsToBeCleared() {
```

Comment:
The name `clearParams` would make more sense since the method actually does the clear. The current name would indicate that it returns the list of params to be cleared.

Reply:
sure